opencensus/metrics_quickstart/main.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// metrics_quickstart is an example of exporting a custom metric from
+// OpenCensus to Stackdriver.
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"contrib.go.opencensus.io/exporter/stackdriver"
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"golang.org/x/exp/rand"
+)
+
+var (
+	// The latency in milliseconds.
+	latencyMs = stats.Float64("demo/latency", "The latency in milliseconds", "ms")
+
+	latencyView = &view.View{
+		Name:        "demo/latency",
```

Comment:
The name of the metric should represent what latency this is? Request latency, task latency, etc?

Reply:
Done.